packages/opentelemetry-plugin-document-load/src/documentLoad.ts

```diff
@@ -76,12 +84,17 @@ export class DocumentLoad extends BasePlugin<unknown> {
    * Collects information about performance and creates appropriate spans
    */
   private _collectPerformance() {
+    const windowWithTrace: WindowWithTrace = (window as unknown) as WindowWithTrace;
+    const serverContext =
+      parseTraceParent(windowWithTrace.traceparent || '') || undefined;
```

Comment:
Given that `parseTraceParent` already will return `null` if it fails to parse, would this still work without the `|| undefined`?

Reply:
the `undefined` must be there as the `SpanOptions.parent` is optional and cannot be null so the typescript is complaining about it.
https://github.com/open-telemetry/opentelemetry-js/blob/039206493564b4e5858dc883467686385e04317f/packages/opentelemetry-types/src/trace/SpanOptions.ts#L49